pkg/apps/controller/deployer/deployer_controller.go

```diff
@@ -375,6 +389,7 @@ func (c *DeploymentController) makeDeployerPod(deployment *v1.ReplicationControl
 
 	gracePeriod := int64(10)
 
+	t := true
```

Comment:
I wish there is BoolPtr() helper ;-)

Reply:
Please create one and don't use `t` as a variable name, it's hard to follow what's it's about.